src/include/elektra/kdb.h.in

```diff
@@ -38,7 +38,7 @@
 extern "C" {
 #endif
 
-#include "kdbos.h"
+#include <elektra/kdbos.h>
```

Comment:
I'm guessing that's the issue here. If you undo this change it probably works. The include paths are probably not correctly set everywhere in CMake yet. With `"kdbos.h"` the compiler will first look in the same folder as this header, which would work.

Reply:
I implemented the change, but now all builds fail.
This leads me to the conclusion that we do have to fix the CMakeFile instead.
I will rebase to remove the latest change and then try working on the CMakeFile.

Reply:
The reason why this didn't work is probably because the `kdb.h` is generated from the `kdb.h.in` and saved in the build folder instead of the original source folder, so it's not in the same folder as `kdbos.h`